pkg/monitoring/configmaps.go

```diff
@@ -8,10 +8,11 @@ import (
 	"go.uber.org/zap"
```

Comment:
Update copyright `2020, 2021,`

Reply:
ah, yes

Reply:
Addressed in commit e0cb7e9523fa6287222cc134a03a69a0cfd26fa2